<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttentionDialog</class>
 <widget class="QDialog" name="AttentionDialog">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" name="infoLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="sourceLabel">
       <property name="alignment">
        <enum>Qt::AlignRight | Qt::AlignVCenter</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="targetLabel">
       <property name="alignment">
        <enum>Qt::AlignRight | Qt::AlignVCenter</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sourceInfo">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="targetInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skipButton">
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="renameButton">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="retryButton">
       <property name="text">
        <string>Retry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="allBox">
     <property name="text">
      <string>Apply to all items</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
